dev/io.openliberty.microprofile.openapi.internal.common/resources/io/openliberty/microprofile/openapi/internal/resources/OpenAPI.nlsprops

```diff
@@ -98,3 +98,43 @@ OPENAPI_MERGE_INVALID_NAME_CWWKO1666W.useraction=Review the error text to identi
 OPENAPI_MERGE_UNUSED_INCLUDE_CWWKO1667W=CWWKO1667W: The {0} configuration property includes "{1}" but that does not match any deployed application or web module.
 OPENAPI_MERGE_UNUSED_INCLUDE_CWWKO1667W.explanation=The server is configured to create OpenAPI documentation for the application or module name, but no application or web module with that name was deployed.
 OPENAPI_MERGE_UNUSED_INCLUDE_CWWKO1667W.useraction=Check that all applications started correctly and that the names in the configuration property are specified correctly.
+
+OPEN_API_UI_PATH_UPDATE_CWWKO1668I=CWWKO1668I: The OpenAPI UI path has been set to {0}.
+OPEN_API_UI_PATH_UPDATE_CWWKO1668I.explanation=The OpenAPI UI path has been set to a value that is not the default.
+OPEN_API_UI_PATH_UPDATE_CWWKO1668I.useraction=No action is required.
+
+OPEN_API_DOC_PATH_UPDATE_CWWKO1669I=CWWKO1669I: The OpenAPI Document path has been set to {0}.
+OPEN_API_DOC_PATH_UPDATE_CWWKO1669I.explanation=The OpenAPI Document path has been set to a value that is not the default.
+OPEN_API_DOC_PATH_UPDATE_CWWKO1669I.useraction=No action is required.
+
+OPEN_API_UI_PATH_UPDATE_FAILED_CWWKO1670E=CWWKO1670E: The OpenAPI UI path is invalid, the default value will be used instead. Check previous warnings for cause of validation failure.
+OPEN_API_UI_PATH_UPDATE_FAILED_CWWKO1670E.explanation= The OpenAPI UI path is invalid and will use the default value instead.
+OPEN_API_UI_PATH_UPDATE_FAILED_CWWKO1670E.useraction=Check earlier error messages for causes of validation failures.
+
+OPEN_API_DOC_PATH_UPDATE_FAILED_CWWKO1671E=CWWKO1671E: The OpenAPI Document path is invalid, the default value will be used instead. Check previous warnings for cause of validation failure.
+OPEN_API_DOC_PATH_UPDATE_FAILED_CWWKO1671E.explanation= The OpenAPI Document path is invalid and will use the default value instead.
+OPEN_API_DOC_PATH_UPDATE_FAILED_CWWKO1671E.useraction=Check earlier error messages for causes of validation failures.
+
+OPEN_API_UI_PATH_CONFLICT_CWWKO1672E=CWWKO1672E: The OpenAPI UI path cannot be set to {0}. This path is used to retrieve the OpenAPI documents.
+OPEN_API_UI_PATH_CONFLICT_CWWKO1672E.explanation=The OpenAPI Document and OpenAPI UI paths cannot be the same.
+OPEN_API_UI_PATH_CONFLICT_CWWKO1672E.useraction=Specify a unique path for the OpenAPI UI.
+
+OPEN_API_UI_PATH_POTENTIAL_CONFLICT_CWWKO1673W=CWWKO1673W: The OpenAPI UI path was set to the same path as the {0} feature, which might prevent startup of itself or the other feature.
```

Comment:
L2 suggestion to modify word choice for ease of reading:
`OPEN_API_UI_PATH_POTENTIAL_CONFLICT_CWWKO1673W=CWWKO1673W: The OpenAPI UI path was set to the same path as the {0} feature, which might prevent the startup of itself or other features.`

Reply:
ID review:  Either "prevent startup" or "prevent the startup" is acceptable. I prefer "prevent startup"

Reply:
Leaving this as-is